### ISSUE TITLE
Remember filter settings between LOOT instances

### ISSUE THREAD

**Issue by WrinklyNinja** [loot/loot#427]
This was a feature before v0.7, but it got removed in all the churn. It's been requested again [here](http://forums.bethsoft.com/topic/1516193-rel-loot-thread-18/?p=23950020).

Rather than relying on localStorage like in v0.6, which would involve setting up a cache for CEF to use, it's probably easier to just pass the filter state to C++ when quitting, and store this information in settings.yaml.

**Comment by WrinklyNinja** [comment]
Filter memory for the checkbox filters has been implemented in 1f227ddd5543b84a9718f1c710514d705f7447ba.

Memory could easily be implemented for the content filter, but the conflict filter would be trickier. I've avoided implementing memory for those two filters because I think it would be less helpful and indeed welcome. They're not filters I can see anyone wanting on by default. I'll seek some user feedback on this though.
